tests/lib/pkgdb.sh

```diff
@@ -203,10 +203,10 @@ distro_update_package_db() {
             quiet apt-get update
             ;;
         fedora-*)
-            dnf -y -q upgrade
+            dnf -q check-update
```

Comment:
`dnf -q makecache` is the command you're looking for.

`check-update` prints things out (it's like `apt list --upgradable`).